pkg/util/fedinformer/transform_test.go

```diff
@@ -196,6 +201,7 @@ func TestPodTransformFunc(t *testing.T) {
 					NodeName:       "test",
 					InitContainers: []corev1.Container{{Name: "test"}},
 					Containers:     []corev1.Container{{Name: "test"}},
+					RestartPolicy:  corev1.RestartPolicyAlways,
```

Comment:
Why need this?

Reply:
The former test case mainly want to test whether this method can filter metadata information. When I add the metadata, I found that the expected and wanted are the same. Adding this field mainly wants to verify that this method works.